### ISSUE TITLE
Android crashes with Image and React Native 0.55.3

### ISSUE THREAD

**Issue by RGBboy** [lappalj4/react-native-easy-markdown#17]
Essentially embedding any image in the markdown causes the following error on Android:

```
Unexpected view type nested under text node
```

This stems from the fact that View components are nested within Text components, which is no longer supported on Android.

**Comment by juiceo** [comment]
Hmm, iirc this was an issue before as well, which is why (at least the intention was) to not nest images in Text components anyway. Apparently there are cases where that still happens. 

Could you share the input markdown that's causing this?

**Comment by RGBboy** [comment]
Sure, it is literally just an image:

```md
![Test](https://placehold.it/320x320)
```

When in debug mode I get this:

```
==== LOGGING NODE TREE ===
Utils.js:43 > block_0, NODE TYPE: Text
Utils.js:43 > imageWrapper_0_0, NODE TYPE: undefined
```

**Comment by RGBboy** [comment]
Just had a deeper look and it looks like the problem is coming from the `isTextOnly` method in Utils.js. It seems like line 8:
```
nodes[i].type.hasOwnProperty('displayName'))
```
is returning `false`, causing the method to return `true`. Also looks like ReactNative does not have `displayName` anywhere on the `type` property. It may be better to inspect `name` instead.

**Comment by juiceo** [comment]
Hmm, I'll have a look, that might have changed. Thanks for the details!

**Comment by juiceo** [comment]
I pushed a fix to the master branch (removed the isTextOnly from use entirely). I'm unable to test it as I haven't upgraded any of my apps past RN 0.51, would you mind giving it a go?

**Comment by jussi-laitinen** [comment]
I can confirm this fixes the issue.

**Comment by juiceo** [comment]
Thanks. This fix is now published as 1.2.0 in npm.
